### PR TITLE
End to End testing workflow

### PR DESCRIPTION
**Pull request by Atul-source**
End to End testing workflow for l3afd with chaining & without chaining tests

### CODE REVIEW

File: .github/workflows/ci-e2e.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          sudo systemctl stop ufw
+          sudo apt install -y iproute2
+          sudo apt install git curl hey
+      - name: Checkout repository
+        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744
+      
+      - name: Prep
+        run: |
+          sudo cp -r /home/runner/work/l3afd/l3afd /root
+          sudo git clone -b E2E-TEST https://github.com/Atul-source/l3af-arch.git /root/l3af-arch
```
Comment (sanfern) [inline .github/workflows/ci-e2e.yaml:36]:
```suggestion
          sudo git clone https://github.com/l3af-project/l3af-arch.git /root/l3af-arch
```

**Review by sanfern — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-e2e.yaml (hunk 1 above)
Comment (dthaler) [reply .github/workflows/ci-e2e.yaml:36]: +1

Comment (Atul-source) [reply .github/workflows/ci-e2e.yaml:36]: I will do this changes after other PR on l3af-arch repository merged , otherwise this workflow will fail.

**Review by sanfern — APPROVED** [review]
LGTM

**Review by jniesz — APPROVED** [review]

**Pull request merged**